### DISCUSSION TITLE
No From, To, Cc and other addresses in HTML file

### DISCUSSION THREAD

**Discussion by ekateiva** [themsaid/laravel-mail-preview#105] [Q&A]
I do not get addresses in a `.html` mail file. This is what I see as _MessageInfo_

```
<!--
From:[{}], 
to:[{}], 
reply-to:[{}], 
cc:[], 
bcc:[], 
subject:Test subject
-->
```

When I dump `$message->getOriginalMessage()->getFrom()`, I get:

```
array:1 [
  0 => Symfony\Component\Mime\Address^ {#3012
    -address: "no-reply@example.test"
    -name: "Example test (no-reply)"
  }
]
```

And when it is serialized using `json_encode`, it becomes: `"[{}]"`

Laravel version: `Laravel Framework 10.18.0`

**Comment by rudiedirkx** [comment]
I missed those too. `json_encode()` only includes public properties, so not `Address`'s. I can't image Symfony will implement `JsonSerializable` for this, so this package should do something. Can't be too hard. I'll make a PR soon, if I remember.

**Comment by rudiedirkx** [comment]
I finally did something, yay.
